control/content/index.html

```diff
@@ -1,65 +1,408 @@
 <!DOCTYPE html>
-<html ng-app="contactUsPluginContent">
-<head lang="en">
-    <meta charset="UTF-8">
-
-	<!-- CSS -->
-    <link href="../../../../styles/helper.css" rel="stylesheet">
-
-	<!-- JS -->
+<html ng-app='controllerapp'>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script src="../../../../scripts/buildfire.js"></script>
+
     <script src="../../../../scripts/angular/angular.min.js"></script>
+
+    <script src="../../../../scripts/angular/ui-bootstrap.min.js"></script>
+    <script src="../../../../scripts/jquery/jquery-1.11.2.min.js"></script>
```

Comment:
I required the angular.min.js file for working of angular tasks in our app.But that one preexist in our index.html file in comment section along with jquery-1.11.2.min.js .So by mistake that jquery-1.11.2.min.js  was  also included.I will remove that . Thanks for giving me feedback .But here the problem was working of carousel and modal pop up that was not working at that time.So i had to include external Bootstrap.js file